targets/csharp/templates/PlayFabSettings.cs.ejs

```diff
@@ -24,6 +24,27 @@ namespace PlayFab
         public static string DeveloperSecretKey = null;
         /// <summary> You must set this value for PlayFabSdk to work properly (Found in the Game Manager for your title, at the PlayFab Website) </summary>
         public static string TitleId;
+		/// <summary> Placeholder for the Local Debugging URI. </summary>
```

Comment:
You're using tab characters here.
Please replace all tab characters with 4 spaces.

Reply:
🤦‍♂️ I thought I fixed these, but I guess not. I'll look into finding a way to do a global check on the project so that I don't miss any more of these.